src/load_profile_utils.py

```diff
@@ -0,0 +1,14 @@
+import pandas as pd
+
+
+def power(load_profile, t):

```

Comment:
method to interpolate the value of an arbitrary load profile at time `t`. If the value of `t` actually exists in `load_profile`, return the value. If we are either before or after `load_profile`'s time span, return 0 (no power is then present). If `t` is in between start and end, but we haven't got an exact value for `t`, interpolate (i.e. draw a straight line between the two closest points on the load profile graph, and use this value).